### DISCUSSION TITLE
Welcome to TrialMaker Discussions!

### DISCUSSION THREAD

**Discussion by Willy-Kimura** [Willy-Kimura/TrialMaker.Demo#1] [General]
## 👋 Welcome!
Discussions is an awesome place to have great conversations and build a community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Comment on the most recent features.
  * Engage with other community members.

📢 Feature announcements will also be made public!

**Comment by C1rdec** [comment]
Hey Willy could you create a Nuget Package for `System.Evaluation` or you want to keep both the license generator and the System.Evaluation dll private?

That could give you some visibility for Trial Maker.

**Reply by Willy-Kimura** [reply]
Hey C1rdec, 

Yes that's the idea. I'll be working on a NuGet package for the premium version for clients who already have a license. Until then however, only purchasing clients will be able to access both the premium library and the License Generator utility.

Thanks for the question and suggestions!

**Reply by Willy-Kimura** [reply]
Hi C1rdec,

I'm happy to announce that the premium version is now available as a [NuGet package](https://www.nuget.org/packages/TrialMaker.Licensing.Premium)!
Feel free to check it out.

**Comment by C1rdec** [comment]
Nice! And how do you prevent none premium user to use the Library? 
Do we need to activate something with our license?

I was about to do a release of my Project with a local reference. Is there something I need to do if I change to the Nuget package?

**Reply by Willy-Kimura** [reply]
Definitely. 
You'll only be required to verify your license details (once) if you haven't already activated your license using the License Generator utility.
